src/main/java/use_cases/PostManager.java

```diff
@@ -103,17 +103,15 @@ public void createPost(Post post){
     /**
      * Given a post as parameter, returns the same post
      * that is stored in the PostManager
-     * @param post the post object
+     * @param id the unique identifier of a post
      * @return the post stored in the postManager
      */
-    public Post getSpecificPost(Post post)
+    public Post getSpecificPost(String id)
```

Comment:
Thanks for making this method! I'll (PostController) needs to access all the post contents too so I'm gonna add some methods right now to PostManager if that's ok? for example, getPostOwner and getPostRecipe.